assets/sassFolder/_background.sass

```diff
@@ -0,0 +1,18 @@
+body
+    /* Background, and centering */
+    min-height: 100vh
```

Comment:
Generalna sugestija je da vh treba izbegavati dokle god je moguce. Na mobilinim uredjajima to ne radi bas najbolje. Ako ne moze da se namesti sa 100% onda ok...